scripts/content-modules/cp-pages.sh

```diff
@@ -15,7 +15,7 @@ cp -R $SRC/* $DEST/
 find $DEST/ -name "README.md" -exec sh -c 'f="{}"; mv -- "$f" "${f%README.md}_index.md"' \;
 
 # To exclude a file use, e.g.: -not -path '*/specification/_index.md'
-FILES=$(find $DEST -name "*.md")
+FILES=$(find $DEST -name "*.md" -not -path '*/specification/*/semantic_conventions/*')
```

Comment:
We won't want to exclude anything from the OTel spec because that is likely to break things. Whatever the OTel spec repo decides to do with the semconv pages, we'll publish as is on the repo. For example, OTLP placeholder pages were left in the OTel spec after the move to the OTLP repo.